web-platform/website/src/components/pages/home/AddExtensionToast.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+
+import { Strings } from "../../../resources/Strings";
+import { useAuthentication } from "../../../services/AuthenticationService";
+import { useStudies } from "../../../services/StudiesService";
+import { detectBrowser } from "../../../utils/BrowserDetector";
+import { BrowserType } from "../../../utils/BrowserType";
+import { ToastComponent } from "./study-card/ToastComponent";
+
+const extensionStrings = Strings.components.pages.home.alerts.addExtension;
+
+export function AddExtensionToast() {
+  const [showAddExtensionToast, setShowAddExtenionToast] =
+    useState<boolean>(false);
+  const [browserType] = useState(detectBrowser());
+  const { reloadUser } = useAuthentication();
+  const { installedStudyIds, rallyExtensionStudy } = useStudies();
+
+  useEffect(() => {
+    reloadUser();
```

Comment:
Why do we need to reload user? It should be loaded on the page when you load it? It may be that you are not receiving updates since your dependency list is empty and we are not checking isLoaded here. Does this work:
```
const {isLoaded, installedStudyIds, rallyExtensionStudy} = useStudies();

useEffect(() => {
    if (!isLoaded) {
      return;
    }

    showAddExtensionToast(!installedStudyIds.includes(rallyExtensionStudy?.studyId || ""));
}, [isLoaded, rallyExtensionStudy]);
```

isLoaded in studies tackle isLoaded for users.

Reply:
Not sure if this worked or not since the code is unchanged. I am curious to know the outcome.

Reply:
Also, I haven't pushed any of the changes yet. Will soon.

Reply:
@Carla-Moz I renamed that function in my latest PR to reflect this cause the name was definitely confusing:
https://github.com/mozilla-rally/rally/pull/523/files#diff-b6611c3c01013c0196862994772cc4242c13e35ddf7586aa7c5e43e040f00346R43

Reply:
@aaga That's super helpful! Thanks.